autogen.sh

```diff
@@ -1,10 +1,23 @@
 #!/bin/sh
-
 set -e
+
+oldpwd=$(pwd)
+topdir=$(dirname "$0")
+cd "$topdir"
+
+CURRENT_VERSION_FILE=./build/current-version
+if [ -d .git ] # don't ruin file that might come from build-rpm.sh
+then
+	. ./build/versioning.sh
+	echo $GIT_VER > $CURRENT_VERSION_FILE
+	echo $GIT_REF >> $CURRENT_VERSION_FILE
+fi
+
```

Comment:
I would like to keep `autogen.sh` simple as possible. All this stuff can be done inside configure.ac.